### DISCUSSION TITLE
MacBookpro m3 无法正常使用和翻译

### DISCUSSION THREAD

**Discussion by qiqigou1** [crimx/ext-saladict#2157] [Q&A]
按照步骤下载沙拉查词后，无法正常使用功能，有没有大神能解答一下🥹
<img width="1036" alt="截屏2024-01-29 09 36 51" src="https://github.com/crimx/ext-saladict/assets/149810852/8e15e278-6c6b-466a-b07e-81c0002298f0"><img width="716" alt="截屏2024-01-29 09 37 01" src="https://github.com/crimx/ext-saladict/assets/149810852/b0722222-34ed-4eec-8d94-9c78038c92b2">

**Comment by crimx** [comment]
是这个问题么 
https://github.com/crimx/ext-saladict/issues/1629#issuecomment-1080494638

**Comment by qiqigou1** [comment]
是的，已解决！感谢！



---- 回复的原邮件 ----
发件人 ***@***.***> 发送日期 2024年1月29日 17:10 收件人 ***@***.***> 抄送人 Suheng ***@***.***>,
***@***.***> 主题 Re: [crimx/ext-saladict] MacBookpro m3 无法正常使用和翻译 (Discussion #2157)  
是这个问题么
 #1629 (comment)
